shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -158,6 +158,16 @@ - (void)showShareViewController:(NSString*)content {
   UIActivityViewController* activityViewController =
       [[[UIActivityViewController alloc] initWithActivityItems:itemsToShare
                                          applicationActivities:nil] autorelease];
+
+  if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad) {
+    activityViewController.popoverPresentationController.sourceView = engineViewController.view;
+    activityViewController.popoverPresentationController.sourceRect =
+        CGRectMake(CGRectGetWidth(engineViewController.view.bounds) / 2,
+                   CGRectGetHeight(engineViewController.view.bounds) / 2, 0, 0);
```

Comment:
UIKit uses this rect to tell where the pop over should be presented. This should be the rect of the selected text

Reply:
Yeah, but we would have to pass in the rect size/coords from the framework, and then add logic to make sure that the view is being rendered either above/below/next to the text so that it's not clipping off the screen depending on the position of the text and the current view bounds... which is totally doable but since this is a fatal crash I wanted to get it at least non-crashing as soon as possible

Reply:
Never mind, looks like it handles the positioning by itself, will update this PR

Reply:
Never mind, looks like the positioning is automatically handled 👀 will update this PR